src/components/views/read/cvForm/Constants.tsx

```diff
@@ -0,0 +1,10 @@
+import FirstTemplate, { Props } from './templates/FirstTemplate';
+import { ConstTemplateObj } from './types';
+
+export const TEMPLATES: ConstTemplateObj[] = [
+  {
+    id: 'formalTemplate',
+    name: 'Formal Template',
+    component: FirstTemplate as React.FC<Props>,
```

Comment:
It would be nice if you could avoid casting here. Setting a more appropriate type in `types.ts` would help.